src/Nox.Localization/Nox.Localization.csproj

```diff
@@ -20,6 +20,10 @@
         <PackageIcon>nox.png</PackageIcon>
     </PropertyGroup>
 
+    <ItemGroup>
```

Comment:
I did not verify, but this dependency seems that it should not be needed? can you confirm?